migrations/2_deploy_contracts.js

```diff
@@ -182,22 +218,19 @@ module.exports = async (deployer, network) => {
         config.web3.utils.soliditySha3('contract.name', 'rocketNodeStatus'),
         rocketNodeStatus.address
       );
+      await rocketStorageInstance.setString(
+        config.web3.utils.soliditySha3('contract.abi', 'rocketNodeStatus'),
+        compressAbi(rocketNodeStatus.abi)
+      );
       // Log it
       console.log('\x1b[33m%s\x1b[0m:', 'Set Storage RocketNodeStatus Address');
       console.log(rocketNodeStatus.address);
 
-      // Rocket Pool Mini Delegate
-      await rocketStorageInstance.setAddress(
-        config.web3.utils.soliditySha3('contract.address', rocketPoolMiniDelegate.address),
-        rocketPoolMiniDelegate.address
+      // Rocket Pool Mini
```

Comment:
I don't think we'll need to deploy RocketPoolMini. This is a new contract created by RocketFactory for every new minipool, so we won't need to deploy it in the initial migrations.

Reply:
Ah wait, we will need the ABI in there, maybe just add that but not deploy the contract.